hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonTableInputFormat.java

```diff
@@ -804,7 +804,7 @@ public QueryModel getQueryModel(InputSplit inputSplit, TaskAttemptContext taskAt
     Expression filter = getFilterPredicates(configuration);
     boolean[] isFilterDimensions = new boolean[carbonTable.getDimensionOrdinalMax()];
     boolean[] isFilterMeasures =
-        new boolean[carbonTable.getNumberOfMeasures(carbonTable.getTableName())];
```

Comment:
Can u please add comment why you are calling getAllMeasures() instead of getNumberOfMeasures(carbonTable.getTableName())

Reply:
getNumberOfMeasures doesn't include column count which was dropped but it exists in measure list with invisible true whereas the getAllmeasures also include the recent dropped column as well.

Reply:
Please add comment in the code :)

Reply:
done please review.